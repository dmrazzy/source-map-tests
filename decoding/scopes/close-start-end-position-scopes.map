{
  "version": 3,
  "file": "close-start-end-position-scopes.js",
  "sources": [
    "original_0.js"
  ],
  "mappings": "",
  "names": [],
  "scopes": "BAAA,BAAA,CAB,BAAA,CAB,BAAB,CAC,CBA"
}