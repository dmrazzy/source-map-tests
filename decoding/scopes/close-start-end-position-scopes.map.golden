{
  "file": "close-start-end-position-scopes.js",
  "mappings": [],
  "sources": [
    {
      "url": "original_0.js",
      "content": null,
      "ignored": false,
      "scope": {
        "start": {
          "line": 0,
          "column": 0
        },
        "end": {
          "line": 1,
          "column": 0
        },
        "name": null,
        "kind": null,
        "isStackFrame": false,
        "variables": [],
        "children": [
          {
            "start": {
              "line": 0,
              "column": 0
            },
            "end": {
              "line": 0,
              "column": 1
            },
            "name": null,
            "kind": null,
            "isStackFrame": false,
            "variables": [],
            "children": []
          },
          {
            "start": {
              "line": 0,
              "column": 1
            },
            "end": {
              "line": 0,
              "column": 2
            },
            "name": null,
            "kind": null,
            "isStackFrame": false,
            "variables": [],
            "children": []
          },
          {
            "start": {
              "line": 0,
              "column": 3
            },
            "end": {
              "line": 0,
              "column": 5
            },
            "name": null,
            "kind": null,
            "isStackFrame": false,
            "variables": [],
            "children": []
          }
        ]
      }
    }
  ],
  "ranges": []
}